app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+	
```

Comment:
Please check your indentations in your PR and make sure they are correct before sending it.